test-cases/gutenberg/group.md

```diff
@@ -0,0 +1,333 @@
+
+Expected look:  
+![InnerSeparatorBefore](../resources/group-inner-separator-before.png)
+
+--------------------------------------------------------------------------------
+
+##### TC008
+
+### Nested block have proper border styling
+
+-   Add a `Group` block
+-   Nest some blocks inside
+-   Check if each selected block gets solid blue border
+-   Check if each first child gets dashed grey border
+-   Check if margins and paddings do not bounce when changing selection (exception is the trasition of the selected block connected with making space for FloatingToolbar)
+-   Check if there is no weird/unexpected extra free space around blocks
+
+Expected look:  
+![BorderStyling](../resources/group-border-styling.png)
+
+--------------------------------------------------------------------------------
+
+##### TC009
+
+### Nested empty group block do not multiplicates margins
+
+-   Add a `Group` block
+-   Nest multiple empty `Group` block
+-   Check if there is no extra margins applied when nested empty `Group` blocks
+
+Expected look:  
+![PlaceholderNested](../resources/group-placeholder-nest-1.png)
+
+--------------------------------------------------------------------------------
+
+##### TC010
+
+### Nested selection cause applying dimmed style on the rest of blocks
+
+-   Add a `Group` block
+-   Create some nested structure
+-   Navigate down in the hierarchy
+-   Check if each time you change selection all blocks that are higher in hierarchy tree gets dimmed
+-   Select `Group` block
+-   Check if none block gets dimmed
+
+Expected look:  
+![Dimming1](../resources/group-dimming-1.png)
+  
+![Dimming2](../resources/group-dimming-2.png)
+  
+![Dimming3](../resources/group-dimming-3.png)
+
+--------------------------------------------------------------------------------
+
+##### TC011
+
+### No extra margins applied when nesting empty group blocks
+
+-   Add new post
+-   Add a `Group` block
+-   Create some nested structure
+-   Select nested block to make some block gets dimmed
+-   Close post
+-   Reopen post
+-   Check if there is no block beeing selected and dimmed
+
+--------------------------------------------------------------------------------
+
+##### TC012
+
+### FloatingToolbar is render when group or it's child is selected
+
+-   Add a `Group` block
+-   Prepare nested structure
+-   Check if each time you select any block it renders `FloatingToolbar` above
+
+Expected look:  
+![FloatingToolbarRoot](../resources/group-floating-toolbar-root.png)
+  
+![FloatingToolbarNested](../resources/group-floating-toolbar-nested.png)
+
+--------------------------------------------------------------------------------
+
+##### TC013
+
+### Breadcrumbs on FloatingToolbar is properly displayed
+
+-   Add a `Group` block
+-   Create nested structure
+-   Check if each time you change selection `Breadcrumbs` show proper block title and icon 
+-   Check if each time you change selection `Breadcrumbs` show proper parent block
```

Comment:
Let's avoid using subjective descriptions as "proper" word and instead try to use more specific words.
"parent block icon -> focused block icon + focused block title"

Reply:
just realized, "parent block" can also be vague. "top most parent" is better

Reply:
ok, fixed